docs/new-architecture-app-intro.md

```diff
@@ -13,21 +13,29 @@ There are a few prerequisites that should be addressed before the New Architectu
 
 React Native released the support for the New Architecture with the release `0.68.0`.
 
-This guide is written with the expectation that you’re using the latest React Native release. At the moment of writing, this is `0.70.0`. Besides this guide, you can leverage the [upgrade helper](https://react-native-community.github.io/upgrade-helper/) to determine what other changes may be required for your project.
+This guide is written with the expectation that you’re using the latest React Native release. At the moment of writing, this is `0.71.0`. Beside this guide, you can leverage the [upgrade helper](https://react-native-community.github.io/upgrade-helper/) to determine what other changes may be required for your project.
 
 To update to the most recent version of React Native, you can run this command:
 
 ```bash
-yarn add react-native@0.70.0
+npx react-native upgrade
```

Comment:
nit: this should be backported to all the versions

Reply:
Ops... adressed in #3312